corehq/apps/app_manager/static/app_manager/js/app_logos.js

```diff
@@ -0,0 +1,60 @@
+hqDefine("app_manager/js/app_logos.js", function() {
+    var manager = function() {
```

Comment:
This closure seems unnecessary since the `hqDefine` is already a closure and this one is called exactly once immediately after it is defined. When removing it, I think the only change needed is to do `var manager = self = {};` instead of `var self = this;`